recipes/out_ptr/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  'ea379b2':
```

Comment:
It can not be the commit, use the custom CCI format instead: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases